syntax = "proto3";

package mprpc;
option cc_generic_services = true;

message ResultCode{
    int32 errCode = 1;
    bytes errMsg = 2;
}

message LoginRequest{
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse{
    ResultCode result = 1;
    bool success = 2;
}

service UserServiceRpc{
    rpc login(LoginRequest) returns(LoginResponse);
}